### DISCUSSION TITLE
ERROR: Could not install packages due to an OSError: [Errno 28] No space left

### DISCUSSION THREAD

**Discussion by masettizan** [openai/whisper#1860] [Q&A] [answered]
Hi,
I'm running into this issue while trying to pip install -U openai-whisper. I think it is maybe because I interrupted the first download of it to switch which directory i was in. But I am unsure how to fix this.

I cleared my recently deleted operations that were still running, I checked the other post on here that had a similar error and tried the answer there but it didn't work for me, I checked that I still have inodes left. And I don't know what to do to fix this. But I keep getting the error that I don't have any space left or that.

**Comment by glangford** [comment]
This might be helpful - if not, post a log showing the output from `pip install`

https://askubuntu.com/questions/1422571/errno-28-no-space-left-on-device-even-though-there-is-space

**Reply by masettizan** [reply]
That is helping I think that is helping, tho I might have to delete more. Do you know how big whisper is? cause I freed up like 5gb and it loads way longer than it did before but still ends up failing.

**Reply by glangford** [reply, marked as answer]
Not sure how big it is in total, `pip` will load all of the dependencies as well if they aren't already there so there is more than just whisper itself. Then once you have whisper installed it will need a model (see the README for the various sizes).

**Reply by masettizan** [reply]
I keep only being able to get this far with the download. I have 4.4 gb free and don't know what else to do cause idk how to get even more space.
![image](https://github.com/openai/whisper/assets/117392565/7e322ad1-3f14-4827-a427-9bd7c212a868)

**Reply by glangford** [reply]
If you aren't able to get more resources there, how about using a cloud service (such as Google Colab) or the online version of whisper? You can get your feet wet that way and experiment without needing a lot of local space.

https://platform.openai.com/docs/guides/speech-to-text

**Reply by masettizan** [reply]
There is an online version of whisper? do you know where I could find it, as that would probably be the easiest soultion for me

**Reply by glangford** [reply]
See the link I sent above - all of the OpenAI API's are described here

https://platform.openai.com/docs/introduction

**Comment by vinaypri** [comment]
The solution to this is to install pytorch separately as that consumes the max resources. If you are running on a CPU (non GPU) machine then install pytorch corresponding to that. Visit https://pytorch.org/get-started/locally/ , find the relevant pytorch installation command and run that before installing openai-whisper. It worked for me on a machine with 1 vCPU, 25 GB HDD and 1 GB RAM
